src/rt/dmain2.d

```diff
@@ -477,6 +474,13 @@ extern (C) int _d_run_main(int argc, char **argv, MainFunc mainFunc)
             result = (result == EXIT_SUCCESS) ? EXIT_FAILURE : result;
     }
 
+    import rt.config : rt_configOption;
+
+    if (rt_configOption("noTrapExceptions") !is null)
```

Comment:
Certainly not desirable :) It would mean that programs will segfault instead of showing an exception stack trace on uncaught exceptions by default.